src/app/shared/components/dashboard/profile/profile.component.scss

```diff
@@ -0,0 +1,24 @@
+:host {
+
+  .profile {
+    margin-top: 110px;
+    height: 400px;
+    background-color: rgb(155, 157, 157);
+    background-image: url("/assets/brick-wall.png");
+    color: #545149;
```

Comment:
Don't use #hash colors in component's SCSS.
But instead:
1) create _colors.scss in the /src/styles
<img width="587" alt="image" src="https://user-images.githubusercontent.com/11748604/185808457-87dd406a-5420-453d-9b29-b2019260d5e0.png">
2) add color name for this #hash using web-site https://www.color-blindness.com/color-name-hue/ 
example for this color name is: $masala-brown
3) import _colors.scss into your SCSS and use this color by name
<img width="267" alt="image" src="https://user-images.githubusercontent.com/11748604/185808630-8d42223c-53a3-4782-a194-3041a94ceaf2.png">

Reply:
Done 
<img width="345" alt="Screen Shot 2022-08-25 at 12 33 05 PM" src="https://user-images.githubusercontent.com/74135316/186752491-f045334c-6c1b-4b83-8af5-fea4ae6a7379.png">
<img width="449" alt="Screen Shot 2022-08-25 at 12 32 48 PM" src="https://user-images.githubusercontent.com/74135316/186752499-473c10f1-172e-466a-bd4a-68d3c03f1a11.png">